WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -176,6 +176,9 @@ class NotificationsViewController: UITableViewController, UIViewControllerRestor
             setupAppRatings()
             self.showInlinePrompt()
         }
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.2) {
```

Comment:
Can I suggest we move this call into `showNotificationPrimerAlert`, after the `guard`, so we don't need to do this dispatch every time the view appears? If the alert has already been seen we can bail out of that method, otherwise we can dispatch after the timeout.

Perhaps an interim method could be introduced:

```
private func showNotificationPrimerAlertIfNeeded()
        guard !UserDefaults.standard.notificationPrimerAlertWasDisplayed else {
            return
        }

        DispatchQueue.main.asyncAfter(deadline: .now() + 0.2) {
                self.showNotificationPrimerAlert()
        }
}
```

then just call `showNotificationPrimerAlertIfNeeded` from here.

Reply:
Yup! This makes perfect sense, thank you!!